src/Microsoft.Framework.Configuration/ConfigurationSection.cs

```diff
@@ -40,6 +47,21 @@ public string Key
             }
         }
 
+        public string Path
+        {
+            get
+            {
+                if (!string.IsNullOrEmpty(_prefix))
```

Comment:
If we don't need to allow empty prefix in the constructor we can get rid of this.